e2e/tests/company/administrator/settings/workspace.spec.ts

```
@@ -0,0 +1,71 @@
+
+  test("displays initial company data in form fields", async ({ page }) => {
+    const companyNameInput = page.getByLabel("Company name");
+    const websiteInput = page.getByLabel("Company website");
+
+    await expect(companyNameInput).toHaveValue(company.name as string);
+
+    const companyData = await db.query.companies.findFirst({
+      where: eq(companies.id, company.id),
+    });
+
+    await expect(websiteInput).toHaveValue(companyData?.website as string);
```

ESLint

[error] Do not use any type assertions.

(@typescript-eslint/consistent-type-assertions)